docs/auth/social.mdx

```diff
@@ -138,10 +138,10 @@ import 'package:flutter_facebook_auth/flutter_facebook_auth.dart';
 
 Future<UserCredential> signInWithFacebook() async {
   // Trigger the sign-in flow
-  final AccessToken result = await FacebookAuth.instance.login();
+  final LoginResult loginResult = await FacebookAuth.instance.login();
 
   // Create a credential from the access token
-  final facebookAuthCredential = FacebookAuthProvider.credential(result.token);
+  final OAuthCredential facebookAuthCredential = FacebookAuthProvider.credential(loginResult.accessToken.token);
 
   // Once signed in, return the UserCredential
   return await FirebaseAuth.instance.signInWithCredential(facebookAuthCredential);
```

Comment:
Could you remove the `await` keyword?

Reply:
Hello, I updated the place you specified in the "[versions.js](https://github.com/FirebaseExtended/flutterfire/blob/master/docs/versions.js#L44)" file as you said. But when I remove the `await` keyword, the code doesn't work. I didn't quite understand what you mean. Thanks.

Reply:
https://github.com/FirebaseExtended/flutterfire/pull/6719#issue-697367415 this is the pull request.

Reply:
The `await` keyword is unnecessary because the function return value is `Future<UserCredential>`.

The below will return exactly: `Future<UserCredential>`. It will work.
```dart
return FirebaseAuth.instance.signInWithCredential(facebookAuthCredential);
```

Reply:
Done.